frontend/src/components/Hero.tsx

```diff
@@ -8,14 +11,22 @@ const Hero = () => {
 
   return (
     <div data-testid="hero" className="usa-dark-background bg-primary">
-      <GridContainer className="padding-y-1 tablet:padding-y-3 tablet-lg:padding-y-10 desktop-lg:padding-y-15">
+      <GridContainer className="padding-y-1 tablet:padding-y-3 tablet-lg:padding-y-10 desktop-lg:padding-y-15 position-relative">
```

Comment:
Added positioning so that the child link can be positioned relative to this element.